features/step_definitions/table_steps.rb

```diff
@@ -40,7 +40,8 @@
 
   retries ||= 1
 
-  expected.diff! table
+  options = exact_columns.present? ? { surplus_col: true } : {}
```

Comment:
With the `surplus_col` option, the comparison will fail if the actual table has an extra column that is not defined in the expected table.